submissions/victorAienobe/css/contact.css

```diff
@@ -0,0 +1,203 @@
+body {
+    width: 100%;
+    background-color: #ffffff;
+    margin:0;
+    padding:0;
+    height: auto;
+}
+.pic {
+    padding-left: 20px;
+}
+.top {
+    width: 100%;
+    height: 70px;
+    background-color: purple;
+}
+.nav-link {
+    height: 19px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 15px;
+    line-height: 1.19;
+    letter-spacing: normal;
+    color: #ffffff;
+    text-decoration: none;
+    float: right;
+    margin-right: 40px;
+    margin-top: 25px;  
+}
+.main {
+    display: flex;
+    flex-direction: row;
+    width: 90%;
+    margin: 30px 30px 30px 150px; 
+    padding: 30px 0;
+} 
+.main1 {
+    text-align: center; 
+    padding-top: 14px;
+    font-size: 22px;
+    margin-top: 30px;
+    margin-right: 30px;
+}
+aside{
+        
+        height: 490px;
+        width: 35%;
+        margin: auto; 
+        border: solid 1px #707070;
+        background-color: #ffffff;
+        text-align: center;
+        padding: 0px;
+}
+.sub {
+    margin-left: 0 auto;
+}
+#text2 {
+    float: left;
+    margin: 20px 0 0 37px;
+}
+.email {
+    margin: 20px auto;
+}
+.text {
+    text-align: center;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
@ainobs Take this to the body tag so that all the elements on the page should use that font.